acceptance/tests/custom_facts/windows_not_expand_command.rb

```diff
@@ -0,0 +1,34 @@
+test_name 'FACT-2054: Custom facts that execute on windows should raise an error' do
```

Comment:
```suggestion
test_name 'FACT-2054: Execute on Windows with expand => false should raise an error' do
```

a more suggestive test name

Reply:
excelent